src/System.Reflection.Emit.Lightweight/ref/Configurations.props

```diff
@@ -4,6 +4,7 @@
     <BuildConfigurations>
       netcoreapp;
       uap;
+      netstandard;
```

Comment:
You'll need a netfx configuration for all of these that typeforwards so that you don't have duplicate types.

Reply:
This assembly exists inbox on .NET Framework already so I think the placeholder configuration is correct but we should validate that the contract is valid which would happen in the shim apicompat run (which I think are currently manual).